### ISSUE TITLE
artm.Dictionary.filter: max_dictionary_size type error

### ISSUE THREAD

**Issue by Alvant** [bigartm/bigartm#1014]
![wrong_type](https://user-images.githubusercontent.com/15067981/86008190-18253480-ba21-11ea-8ba6-39feef4f5890.png)

**Comment by JeanPaulShapo** [comment]
Sorry for late response.

In fact, there is a bug in docstring: according to [description of FilterDictionaryArgs](https://github.com/bigartm/bigartm/blob/v0.9.2/src/artm/messages.proto#L530) `max_dictionary_size` argument should be integral, not float.
So, right now it's impossible to specify maximum dictionary size as fraction of initial dictionary size, only as absolute number.

**Comment by Alvant** [comment]
Thanks!

If you could point me to the right files, I would be happy to fix the docstring and make a pullrequest as a part of this issue :)

**Comment by JeanPaulShapo** [comment]
Yeah, sure - [link to this typo in master branch](https://github.com/bigartm/bigartm/blob/master/python/artm/master_component.py#L385).
Note: we support two major versions of bigartm library, so you can create PR against `master` branch only - after that I'll backport this fix to branch for `0.9.x `major version, and hopefully we'll ship releases with this fix in the distant future :)

**Comment by Alvant** [comment]
Ok) Done: https://github.com/bigartm/bigartm/pull/1026

**Comment by JeanPaulShapo** [comment]
Thanks! You fix is already in online documentation!
Ahh! I've recenlty found the other part with the same typo in [Dictionary class](https://github.com/bigartm/bigartm/blob/master/python/artm/dictionary.py#L172). Could you make PR to address this bug too?
